src/vulkan-renderer/wrapper/uniform_buffer.cpp

```diff
@@ -4,6 +4,7 @@
 
 #include <cassert>
 #include <cstring>
+#include <utility>
```

Comment:
Why was the `#include <utility>` added? (Seems like it wasn't required before and the changes don't need it.)

(Might also be in more files than commented ones.)

Reply:
https://github.com/inexorgame/vulkan-renderer/issues/287: same as other.